### DISCUSSION TITLE
Delete Local Changes and Revert to origin

### DISCUSSION THREAD

**Discussion by Bacon117** [community/community#40943] [New to GitHub]
Newbie here. 

I want to pull changes from the origin (remote repository) and overwrite (delete) any changes I have made locally.  I'm using GitHub Desktop.  It's not immediately clear to me how to do this, perhaps it's just a terminology thing... 

Thanks in advance.

**Comment by GabyCT** [comment]
Problably you can try this https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/managing-commits/reverting-a-commit

**Comment by airtower-luna** [comment]
The Git term for "get my current branch and working directory to the state of some other (possibly remote) branch, discarding any changes" is *hard reset*. I don't know how to do it in GitHub Desktop, on the command like the command would be something like this (to revert to the state of `origin/main`):

```
git reset --hard origin/main
```

Mind that any locally changed files will be overwritten. :warning:
